.github/ISSUE_TEMPLATE/06-performance.yml

```diff
@@ -0,0 +1,168 @@
+      label: Urgency
+      placeholder: Is your issue urgent? If important use cases are blocked, there is a regression in ONNX Runtime functionality, or you have a strict project deadline, please elaborate here.
+    validations:
+      required: false
+  - type: markdown
+    attributes:
+      value: |
+        # System information
+        For each of the following items, provide the information listed to ensure that your request is addressed in a timely manner.
+  - type: dropdown
+    id: platform
+    attributes:
+      label: Platform
+      options:
+        - Windows
+        - Linux
+        - Mac
+        - Android
+        - iOS
+        - Web Browser
+        - Other / Unknown
+    validations:
+      required: true
+  - type: input
+    id: os-dist
+    attributes:
+      label: OS Distribution
+      placeholder: ex. 16.04
+    validations:
+      required: true
+  - type: dropdown
+    id: ort-installation
+    attributes:
+      label: ONNX Runtime Installation
+      options:
+        - Source
+        - Binary
+        - Other / Unknown
+    validations:
+      required: true
+  - type: input
+    id: ort-version
+    attributes:
+      label: ONNX Runtime Version
+      placeholder: ex. 1.12.1
+    validations:
+      required: true
+  - type: dropdown
+    id: inf-api
+    attributes:
+      label: API
+      options:
+        - Python
+        - C++
+        - C#
+        - C
+        - Java
+        - JavaScript
+        - Obj-C
+        - WinRT
+        - Other / Unknown
+    validations:
+      required: true
+  - type: input
+    id: api-version
+    attributes:
+      label: API Version
+      placeholder: ex. 3.9.13
+    validations:
+      required: true
+  - type: dropdown
+    id: arch
+    attributes:
+      label: Architecture
+      options:
+        - X64
+        - X86
+        - ARM64
+        - ARM32
+        - IBM Power
+        - Other / Unknown
+    validations:
+      required: true
+  - type: dropdown
+    id: hardware-accel
+    attributes:
+      label: Hardware Acceleration
+      options:
+        - Default CPU
+        - CoreML
+        - CUDA
+        - DirectML
+        - oneDNN
+        - OpenVINO
+        - TensorRT
+        - NNAPI
+        - ACL
+        - ArmNN
+        - MIGraphX
+        - Rockchip NPU
+        - SNPE
+        - TVM
+        - Vitis AI
+        - Other / Unknown
+    validations:
+      required: true
+  - type: input
+    id: ha-version
+    attributes:
+      label: Hardware Acceleration Version
+      placeholder: ex. 2022.1
+    validations:
+      required: true
+  - type: dropdown
+    id: model
+    attributes:
+      label: Model Type
+      options:
+        - BERT
+        - GPT2
+        - Hugging Face
+        - Longformer
+        - T5
+        - Other / Unknown
+    validations:
+      required: true
+  - type: input
+    id: vs-version
+    attributes:
+      label: Visual Studio Version (if applicable)
+      placeholder: ex. 17.2.5
+  - type: input

```

Comment:
Compiler and VS versions are only needed for the Build template and can be removed for all others.